src/client/client.cpp

```diff
@@ -1012,6 +1012,13 @@ void Client::ProcessData(NetworkPacket *pkt)
 		return;
 	}
 
+	if (!pkt->getWasReliable() && !toClientCommandTable[command].can_be_unreliable) {
+		infostream << "Client::ProcessData(): Recieved an unreliable packet,"
+			"but we can only process reliable packets for kind of packet: "
+			<< toClientCommandTable[command].name;
```

Comment:
not saying it's better but all other log messages use the number, not the name

Reply:
should I change the message below this one to include the packet name then? Name seems better on comprehensibility, but obviously can't use the name if the packet is unknown.

Reply:
sure